### ISSUE TITLE
add the source path for standardalone computer

### ISSUE THREAD

**Issue by RAJU2529** [MicrosoftDocs/OfficeDocs-DeployOffice#724]
Some windows computer users are not connected to corporate. so in this case . user needs to download to  local computer. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 66127c02-c2dc-f6d1-6e83-a6d2b1166731
* Version Independent ID: 27635315-22db-d136-f07e-5169f6207b50
* Content: [Overview of the Office Deployment Tool](https://docs.microsoft.com/en-us/deployoffice/overview-of-the-office-2016-deployment-tool#feedback)
* Content Source: [DeployOffice/overview-of-the-office-2016-deployment-tool.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/overview-of-the-office-2016-deployment-tool.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @jwhit-MSFT
* Microsoft Alias: **jwhit**

**Comment by davguent** [comment]
@RAJU2529 Can you add some additional detail?  If this is about users who usually are at work and due to COVID-19 etc. are working remote, please see blog I wrote.
Configuring Office 365 ProPlus updates for remote workers using VPN
https://techcommunity.microsoft.com/t5/office-365-blog/configuring-office-365-proplus-updates-for-remote-workers-using/ba-p/1253491

Let us know if this doesn't address concern.

**Comment by RAJU2529** [comment]
@davguent   Jus now I checked your blog. I found ist about vpn connection. Actually in main article, configuration xnl file have source path like \\share\\ofice.

But it does  not for  home users .. I  said add source path  for home users in the article.

Last week I configured configuration file and downloaded office 2019 suite successfully.  Only most advanced users to configure .XML file. but normal users does not know .

**Comment by davguent** [comment]
@RAJU2529 OK, my mistake as blog is about updating Office you're interested in installing Office.  By default, the Setup.exe (ODT) looks in working directory at install time for Office folder.  If not present, it will by default download content from CDN.  Can you rephrase and define the outcome you are looking for remote user.  Example:  They don't have Office or have older version of Office and want to get Office 2019 without corporate network but they do have internet or they don't have corporate .  I need more information to better comment.

**Comment by RAJU2529** [comment]
@davguent . downloaded  ODT have default configutaion file like below

**<Configuration>

  <Add OfficeClientEdition="64" Channel="Monthly">
    <Product ID="O365ProPlusRetail">
      <Language ID="en-us" />
    </Product>
    <Product ID="VisioProRetail">
      <Language ID="en-us" />
    </Product>
  </Add>

  

  

  

</Configuration>**

**Comment by RAJU2529** [comment]
@davguent . I  configured the xml and source path 
[Office365.zip](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/files/4421516/Office365.zip)

**Comment by manoth-msft** [comment]
For standalone computers without corp network and none-corp users, there is no need to set source path. Either you don't donwload the files upfront (they will be streamed down during install) or you download the files and they will be put into the same location as the setup.exe and pciked up from there automatically.

**Comment by manoth-msft** [comment]
@jwhit-MSFT As this is not related to updating, I suggest to close this thread.
